serokell-style.md

```diff
@@ -288,119 +285,117 @@ Prefer two-line imports for such standard containers.
 
 ```haskell
 import Data.Map (Map)
-
-import qualified Data.Map as Map hiding (Map)
+import qualified Data.Map as Map
 ```
 
-Such tools as `stylish-haskell` can make your import section look very nice!
-
 ### Data Declarations
 
-Align the constructors in a data type definition.  Example:
+Align the constructors in a data type definition. Examples:
 
 ```haskell
-data Tree a = Branch !a !(Tree a) !(Tree a)
-            | Leaf
-```
-
-For long type names the following formatting is also acceptable:
+data Tree a
+  = Branch a (Tree a) (Tree a)
+  | Leaf
 
-```haskell
 data HttpException
-    = InvalidStatusCode Int
-    | MissingContentHeader
+  = InvalidStatusCode Int
+  | MissingContentHeader
 ```
 
 Format records as follows:
 
 ```haskell
 data Person = Person
-    { firstName :: !String  -- ^ First name
-    , lastName  :: !String  -- ^ Last name
-    , age       :: !Int     -- ^ Age
-    } deriving (Eq, Show)
+  { firstName :: String  -- ^ First name
+  , lastName  :: String  -- ^ Last name
+  , age       :: Int     -- ^ Age
+  } deriving (Eq, Show)
 ```
 
-If you have record with multiple constructors (which is generally bad idea
-because you getters become partial functions but okay if you use
-`-XRecordWildCards`) then align curly braces with
-shift to the constructor name but } should go in the end of last field.
+You should generally avoid having records with multiple constructors because
```

Comment:
Maybe let's prohibit them completely? If you have `data X = A { aQ :: Q, aW :: W} | B {bQ :: Q, bE :: E}` you can define `data AData = AData {aQ :: Q, aW :: W}` instead (and similar for `B`) and have `data X = A AData | B BData` which is very similar, but safer.